test/widgets.dart

```diff
@@ -203,16 +203,17 @@ class WidgetWithDisposableFeatureState
   Widget build(_) => TestWrapper(
         child: Column(
           children: <Widget>[
-            _showDisposableFeature
-                ? DescribedFeatureOverlay(
-                    featureId: widget.featureId,
-                    allowShowingDuplicate: false,
-                    enablePulsingAnimation: false,
-                    title: Text(widget.disposableFeatureTitle),
-                    tapTarget: Icon(widget.featureIcon),
-                    child: Container(),
-                  )
-                : Container(),
+            if (_showDisposableFeature)
+              DescribedFeatureOverlay(
+                featureId: widget.featureId,
+                allowShowingDuplicate: false,
+                enablePulsingAnimation: false,
+                title: Text(widget.disposableFeatureTitle),
+                tapTarget: Icon(widget.featureIcon),
+                child: Container(),
+              )
+            else
+              Container(),
```

Comment:
I think you can delete the `else` now

Reply:
Right. Pretty amazing that you can finally mark multiple lines - I did not even realize.